### ISSUE TITLE
duplicate registraion is being posted via /register

### ISSUE THREAD

**Issue by Dolar1** [bradtraversy/devcamper-api#16]
@bradtraversy I think in response with the /register route we are letting duplicate registrations save to DB.

It may sound conflicting but check it once.

**Comment by devrinku** [comment]
Yes it happens ,sometimes duplicate error does not arises...And you keep on saving the same thing,that you have marked as unique true in model.....i think its an issue with mongoDb....[https://stackoverflow.com/questions/5535610/mongoose-unique-index-not-working](url)
so you have to manually check those feilds.....
